srcpkgs/distrobox/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Distrobox requires one of 'docker' or 'podman' to be installed.
```

Comment:
The policy on void for things like that is to not have an INSTALL message nor a `README.voidlinux` file, and instead rely on upstream documentation.
The install message is only for breaking changes, and the readme is only for void-specific information.

Distrobox provides clear documentation that one of these container runtimes is needed (including on the homepage), and clear [error messages](https://github.com/89luca89/distrobox/blob/main/distrobox-create#L472-L480) when none is available.